cmd/image-builder/logging.go

```diff
@@ -0,0 +1,66 @@
+package main
```

Comment:
just a nit: should this really live in cmd? shouldn't it live under `internal/logging` or `internal/v1/logging`?

Reply:
Oki :)